quantestpy/visualization/pauli_circuit_drawer.py

```diff
@@ -0,0 +1,336 @@
+            reg_name_max_length += 1
+
+        id_max_length = len(str(self._num_qubit-1))
+        for line_id in range(self._num_line):
+            if line_id in self._line_id_to_qubit_id.keys():
+                qubit_id = self._line_id_to_qubit_id[line_id]
+                reg_name = self._qubit_id_to_output_reg_name[qubit_id]
+                self._line_id_to_text[line_id] += reg_name \
+                    + self.get_space(reg_name_max_length-len(reg_name)) \
+                    + str(qubit_id) \
+                    + self.get_space(id_max_length-len(str(qubit_id)))
+            else:
+                self._line_id_to_text[line_id] += \
+                    self.get_space(id_max_length+reg_name_max_length)
+
+    def draw_init_vector(self) -> None:
+        """Draws initial state vectors.
+        ::
+
+        |1>
+        """
+        for line_id in range(self._num_line):
+            if line_id in self._line_id_to_qubit_id.keys():
+                qubit_id = self._line_id_to_qubit_id[line_id]
+                qubit_val = self._qc.qubit_value[qubit_id]
+                obj = self.get_state(qubit_val)
+            else:
+                obj = self.get_space(length=3)
+            self.add_obj_in_line_id_to_text(line_id, obj)
+
+    def draw_final_vector(self) -> None:
+        """Draws final state vectors.
+        ::
+
+        |0>
+        """
+        self.draw_init_vector()
+
+    def draw_init_phase(self) -> None:
+        """Draws qubit phases in unit of PI.
+        ::
+
+        0.0
+        """
+        phase_max_length = 0
+        for qubit_phase in self._qc.qubit_phase:
+            phase_length = len(self.get_phase(qubit_phase))
+            if phase_length > phase_max_length:
+                phase_max_length = phase_length
+
+        for line_id in range(self._num_line):
+            if line_id in self._line_id_to_qubit_id.keys():
+                qubit_id = self._line_id_to_qubit_id[line_id]
+                qubit_phase = self._qc.qubit_phase[qubit_id]
+                phase = self.get_phase(qubit_phase)
+                obj = phase + self.get_space(phase_max_length-len(phase))
+            else:
+                obj = self.get_space(length=phase_max_length)
+            self.add_obj_in_line_id_to_text(line_id, obj)
+
+    def draw_final_phase(self) -> None:
+        """Draws qubit phases in unit of PI.
+        ::
+
+        0.5
+        """
+        self.draw_init_phase()
+
+    def color_line(self, line_id: int, text: str) -> None:
+        """Colors lines according to their qubit values"""
+        if line_id in self._line_id_to_qubit_id.keys():
+            qubit_id = self._line_id_to_qubit_id[line_id]
+            qubit_val = self._qc.qubit_value[qubit_id]
+            color_code = self.get_color_code_line(qubit_val)
+            obj = self.add_color_code_in_obj(color_code, text)
+            self.add_obj_in_line_id_to_text(line_id, obj, replace_obj=True)
+
+    def draw_one_gate(self, gate_id: int) -> None:
```

Comment:
@mitz1012 
Now the coloring is put in draw_one_gate() method and is factored out from the other draw methods.